src/winconpty/dll/winconptydll.vcxproj

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{a22ec5f6-7851-4b88-ac52-47249d437a52}</ProjectGuid>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>winconpty</RootNamespace>
+    <ProjectName>winconpty.DLL</ProjectName>
+    <TargetName>conpty</TargetName>
+    <ConfigurationType>DynamicLibrary</ConfigurationType>
+  </PropertyGroup>
+  <Import Project="$(SolutionDir)src\common.build.pre.props" />
+  <ItemGroup>
+    <ClCompile Include="../precomp.cpp">
+      <PrecompiledHeader>Create</PrecompiledHeader>
+    </ClCompile>
+    <None Include="winconpty.def" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\lib\winconptylib.vcxproj">
+      <Project>{58a03bb2-df5a-4b66-91a0-7ef3ba01269a}</Project>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemDefinitionGroup>
+    <Link>
+      <ModuleDefinitionFile>winconpty.def</ModuleDefinitionFile>
+    </Link>
+  </ItemDefinitionGroup>
+  <!-- Careful reordering these. Some default props (contained in these files) are order sensitive. -->
+  <Import Project="$(SolutionDir)src\common.build.post.props" />

```

Comment:
would you mind carrying these lines over from the other project just in case _somebody_ gets the bright idea to roll up this dll into a universal project?
It may also help with other packaging formats.

https://github.com/microsoft/terminal/pull/3565/files#diff-02ffd8a49ce6e2af6082ab675019b410L33-L65

Reply:
Sure